src/ethereum-json-rpc-client/src/canister_client.rs

```diff
@@ -52,6 +59,69 @@ impl<T: CanisterClient + Sync + 'static> Client for T {
     }
 }
 
+impl<T: CanisterClient + Sync + 'static> HttpOutcall for T {
```

Comment:
I don't understand this implementation because the way it is implemented does not follow the same design of the other clients.
I would have expected a new HttpOutcallClient in the canister-sdk instead of this

Reply:
I agree, Let me fix it up, Thanks!